ide/static/js/content.js

```diff
@@ -293,6 +369,7 @@ class Content extends React.Component {
       success: function (response) {
         if (response.result === 'success'){
           this.initialiseImportedNet(response.net,response.net_name);
+          this.loadLayerShapes();
```

Comment:
Also, if you are calculating the shapes during import, can you remove the additional calls made to `get_shapes` during export in `export_json`.

Reply:
Yeah Sure